other/upgrade-helper.php

```diff
@@ -318,6 +318,23 @@ function quickFileWritable($file)
 	return is_writable($file);
 }
 
+/**
+ * Delete a file.  Check permissions first, just in case.
+ *
+ * @param string $file
+ */
+function deleteFile($file)
+{
+	if (!file_exists($file))
+		return;
+
+	quickFileWritable($file);
+
+	@unlink($file);
```

Comment:
Should the error    suppression be removed here?

Reply:
Probably not.